### PR TITLE
Feature/bancor ws service

### PR DESCRIPTION
**Pull request by dmitrylineup**
bancor data's consumption via web sockets was added

### CODE REVIEW

File: src/app/data/models/bancor-dto.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+    typeToString(): string {
+        switch (this.type) {
+            case 'BUY':
+                return 'Buy';
+            case 'SELL':
+                return 'Sell';
+            case 'ADD':
+                return 'Add liquidity';
+            case 'REM':
```
Comment (makspotter) [inline src/app/data/models/bancor-dto.ts:61]: Create enum from this props: BUY, SELL, ADD, REM  in separated file and move it shared folder.
Then use enums here and in other files

File: src/app/data/services/data/uniswap-data.service.ts [hunk 2]
```diff
@@ -60,6 +65,30 @@ export class UniswapDataService {
     this.snack.openSnack(Object.assign(new UniswapDto(), uniDto)?.print());
   }
 
+    private handleFarmTradeBancor(bancorDto: BancorDto): void {
+        if (!bancorDto
+            || bancorDto?.coinAddress !== Addresses.ADDRESSES.get('FARM')
+            || bancorDto?.type === 'REM'
+            || bancorDto?.type === 'ADD'
```
Comment (makspotter) [inline src/app/data/services/data/uniswap-data.service.ts:72]: REM, ADD - from enum (see prev. comment)

File: src/app/data/static/utils.ts [hunk 3]
```diff
@@ -209,4 +209,16 @@ export class Utils {
       return 'https://etherscan.io';
     }
   }
+
+  public static prettyTransactionType(type: string): string {
+      const TransactionType = {
+                              'BUY': 'But',
```
Comment (makspotter) [inline src/app/data/static/utils.ts:215]: Typo in `But`

File: src/app/data/static/utils.ts [hunk 4]
```diff
@@ -209,4 +209,16 @@ export class Utils {
       return 'https://etherscan.io';
     }
   }
+
+  public static prettyTransactionType(type: string): string {
```
Comment (makspotter) [inline src/app/data/static/utils.ts:213]: When I said about enum I mean you must do this like this:
`transactions-types.constant.ts`:
```
export enum TransactionsTypes {
  BUY: 'Buy',
  SELL: 'Sell',
  ADD: 'Add liquidity',
  REM: 'Remove liquidity',
  CONVERSION: 'Conversion',
}
```

then you can use it like this:
```
  || uniDto?.type === TransactionsTypes.REM
  || uniDto?.type === TransactionsTypes.ADD
```
at etc.

**Review by makspotter — APPROVED** [review]

**Comment by EmreKspToken** [comment]
EmreKspToken

**Pull request merged**